website/pages/en/new-chain-integration.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Integrating New Networks
+---
+
+Support for new on-chain data sources can be added to Graph Node, allowing developers to build and deploy subgraphs and substreams-based subgraphs for new chains. Two different interfaces can be used for blockchain data ingestion through Graph Node: EVM JSON-RPC and Firehose.
```

Comment:
I think this creates the wrong mental model for people.

Graph Node can currently index data from the following chain types:
Ethereum, via EVM JSON-RPC and [Ethereum Firehose](https://github.com/streamingfast/firehose-ethereum)
NEAR, via a [NEAR Firehose](https://github.com/streamingfast/near-firehose-indexer)
Cosmos, via a [Cosmos Firehose](https://github.com/graphprotocol/firehose-cosmos)
Arweave, via an [Arweave Firehose](https://github.com/graphprotocol/firehose-arweave)

If you are interested in any of those chains, integration is a matter of Graph Node configuration and testing.

If you are interested in a different chain type, you will need to build a new integration with Graph Node. Our recommended approach is development of a Firehose for the chain, and then integration of that Firehose with Graph Node.

Reply:
good point. I can ensure people know the existing _chain type_ . I'll update this